packages/p/pixman/package.yml

```diff
@@ -1,8 +1,8 @@
 name       : pixman
-version    : 0.44.2
-release    : 21
+version    : 0.46.0
+release    : 22
 source     :
-    - https://www.x.org/releases/individual/lib/pixman-0.44.2.tar.xz : 50baf820dde0c5ff9714d03d2df4970f606a3d3b1024f5404c0398a9821cc4b0
+    - https://cairographics.org/releases/pixman-0.46.0.tar.xz : d2eab57e1ce79de991f8ceb3fcd726a6978b970382c8ac8c8f112b61ceaa9167
```

Comment:
What's with the change in the domain name?

Reply:
I did not check that before updating. I assumed that since pixman was created by cairo that it would be under cairographics.